src/components/templates/PartyMap/components/Map/Map.scss

```diff
@@ -58,6 +58,7 @@ $border-radius: 28px;
   .seat-column {
     width: 100%;
     height: 100%;
+    z-index: z(map-seat);
```

Comment:
Is there any reason this needs to be applied to the seat? I would have expected to apply it to the grid itself?

Reply:
There is no grid wrapper

![image](https://user-images.githubusercontent.com/63194656/110914933-b7742880-831f-11eb-982d-796ecc164a5f.png)

Reply:
Fair, that makes sense. No changes required here then.